docs/types.rst

```diff
@@ -826,22 +842,31 @@ shown in the following example:
     }
 
 The contract does not provide the full functionality of a crowdfunding
-contract, but it contains the basic concepts necessary to understand structs.
-Struct types can be used inside mappings and arrays and they can itself
-contain mappings and arrays.
+contract, but it contains the basic concepts necessary to understand structs. Struct types can be used inside mappings and arrays and they can itself contain mappings and arrays.
+
+It is not possible for a struct to contain a member of its own type, although the struct itself can be the value type of a mapping member. This restriction is necessary, as the size of the struct has to be finite.
+
+...note::
+    All the functions above assign a ``struct`` type to a local variable (of the default storage data location). This does not copy the ``struct`` but only stores a reference so that assignments to members of the local variable actually write to the state.
 
-It is not possible for a struct to contain a member of its own type,
-although the struct itself can be the value type of a mapping member.
-This restriction is necessary, as the size of the struct has to be finite.
+If you copy a struct over another struct, for example.
```

Comment:
Please don't break up the sentence like this.

Reply:
@chriseth 

As in the 

> This restriction is necessary, as the size of the struct has to be finite.

part? Sorry the diff highlighted is slightly confusing, and I'm not sure it's showing the change I actually made in context.

Reply:
I meant "If you copy a struct..." is not a full sentence and thus should be part of the previous.